test/test_geometry.py

```diff
@@ -153,3 +155,231 @@ def test_pole_point_inside_polygon_from_vertice_cross(self):
+                    w2_new.tolist())
+
+                # Adjust latitude boundaries to avoid error accumulation
+                if i == 0:
+                    w1_lonlat[1] = b_lonlat[1]
+                elif i >= 9:
+                    w2_lonlat[1] = c_lonlat[1]
+
+                # Update minimum latitude and section if needed
+                min_lat = min(min_lat, w1_lonlat[1], w2_lonlat[1])
+                if np.abs(w2_lonlat[1] -
+                          w1_lonlat[1]) <= ERROR_TOLERANCE or w1_lonlat[
+                              1] == min_lat == w2_lonlat[1]:
+                    min_section = [w1_new, w2_new]
+                    break
+                if np.abs(min_lat - w1_lonlat[1]) <= ERROR_TOLERANCE:
+                    min_section = [w1_new, w2_new] if i != 0 else [v_b, w2_new]
+                elif np.abs(min_lat - w2_lonlat[1]) <= ERROR_TOLERANCE:
+                    min_section = [w1_new, w2_new] if i != 9 else [w1_new, v_c]
+
+            # Update longitude and latitude for the next iteration
+            b_lonlat = ux.grid.coordinates.node_xyz_to_lonlat_rad(
+                min_section[0].tolist())
+            c_lonlat = ux.grid.coordinates.node_xyz_to_lonlat_rad(
+                min_section[1].tolist())
+
+        return np.average([b_lonlat[1], c_lonlat[1]])
+
+    def test_extreme_gca_latitude_max(self):
+        # Define a great circle arc that is symmetrical around 0 degrees longitude
+        gca_cart = np.array([
+            ux.grid.coordinates.normalize_in_place([0.5, 0.5, 0.5]),
+            ux.grid.coordinates.normalize_in_place([-0.5, 0.5, 0.5])
+        ])
+
+        # Calculate the maximum latitude
+        max_latitude = ux.grid.lines.extreme_gca_latitude(gca_cart, 'max')
+
+        # Check if the maximum latitude is correct
+        expected_max_latitude = self._max_latitude_rad_iterative(gca_cart)
+        self.assertAlmostEqual(max_latitude,
+                               expected_max_latitude,
+                               delta=ERROR_TOLERANCE)
+
+        # Define a great circle arc in 3D space
+        gca_cart = np.array([[0.0, 0.0, 1.0], [1.0, 0.0, 0.0]])
+
+        # Calculate the maximum latitude
+        max_latitude = ux.grid.lines.extreme_gca_latitude(gca_cart, 'max')
+
+        # Check if the maximum latitude is correct
+        expected_max_latitude = np.pi / 2  # 90 degrees in radians
+        self.assertAlmostEqual(max_latitude,
+                               expected_max_latitude,
+                               delta=ERROR_TOLERANCE)
+
+    def test_extreme_gca_latitude_min(self):
+        # Define a great circle arc that is symmetrical around 0 degrees longitude
+        gca_cart = np.array([
+            ux.grid.coordinates.normalize_in_place([0.5, 0.5, -0.5]),
+            ux.grid.coordinates.normalize_in_place([-0.5, 0.5, -0.5])
+        ])
+
+        # Calculate the maximum latitude
+        min_latitude = ux.grid.lines.extreme_gca_latitude(gca_cart, 'min')
```

Comment:
did you mean `min` - # Check if the maximum latitude is correct

`_min_latitude_rad_iterative `

Reply:
Yes